content/en/network_monitoring/devices/integrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: NDM Supported Integrations
+description: Learn which integrations are supported by Network Device Monitoring (NDM)
+further_reading:
+    - link: '/network_monitoring/devices/data'
+      tag: 'doc'
+      text: 'SNMP Metrics Reference'
+    - link: 'https://www.datadoghq.com/blog/monitor-meraki/'
+      tag: 'Blog'
+      text: 'Monitor Cisco Meraki with Datadog'
+    - link: 'https://www.datadoghq.com/architecture/network-observability-sd-wan-reference-architecture/'
+      tag: 'Architecture Center'
+      text: 'SD-WAN Reference Architecture'
+
```

Comment:
```suggestion
aliases:
    - /network_monitoring/devices/sd-wan
```

Reply:
There doesn't seem to be a redirect in place for the deleted page, so I'm adding it here. Feel free to move it to another location if this isn't the right place for it.